### PR TITLE
add a way for apps to hook into a page loaded hook

### PR DESCRIPTION
**Pull request by andrewroth**
@twinge 

I'm planning on using this to set up any custom js handlers I need in the form engine (ex. for showing the appropriate debt text, and setting up js handlers for the project preferences options to update when the region of intereste changes)

### CODE REVIEW

File: app/assets/javascripts/fe/fe.public.js [hunk 1]
```diff
@@ -105,6 +105,7 @@
 			$('#page_ajax_spinner').hide();
 			$('.reference_send_invite').button();
 			updateTotals();
+			$(document).trigger('fePageLoaded'); // allow other code to handle page load event by using $(document).on('pageLoaded', function() { ... });
```
Comment (twinge) [inline app/assets/javascripts/fe/fe.public.js]: The comment doesn't match the code (fePageLoaded vs pageLoaded)

**Comment by twinge** [comment]
:+1:

File: app/assets/javascripts/fe/fe.public.js (hunk 1 above)
Comment (andrewroth) [inline app/assets/javascripts/fe/fe.public.js]: Ah, thanks.  I was trying to use $.fe.trigger('pageLoaded') but it seems to only work on elements, and $(document).trigger('...') seems to be standard as it's in an example at the end of https://learn.jquery.com/events/introduction-to-custom-events/

**Comment by jbirdjavi** [comment]
@andrewroth Did you intend to merge this instead of close it?

**Comment by andrewroth** [comment]
Odd, I thought I merged it into the fe branch then delete this branch.  Maybe that automatically closes all pull requests on the branch?  Actually, maybe I merged it on my local computer, deleted the github branch, then pushed to get the merge in and that caused this issue to be closed?

**Comment by andrewroth** [comment]
It's in fe here: https://github.com/CruGlobal/qe/commit/6172d2838f82451a9b169dcdd2682e8f10d1e44a

**Pull request closed without merging**